app/download_manager.py

```diff
@@ -97,7 +99,7 @@ def progress(s):
                     s = s.split(' / ')
                     total = float(s[1])
                     current = float(s[0].split(': ')[1])
-                    progress = int((current / total) * 1000)
+                    progress = int((old_div(current, total)) * 1000)
```

Comment:
Do we need integer division here? At first glance I suspect that standard division is fine and that this is just an automatic change. If we don't need it we should keep the original form for simplicity and drop the extra import.